<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup filter</string>
  </property>
  <property name="windowIcon">
   <iconset theme="edit-find">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="simpleTab">
      <attribute name="title">
       <string>Fields</string>
      </attribute>
      <layout class="QGridLayout" name="fieldsLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="simpleScrool">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="simpleAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>739</width>
            <height>435</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="simpleLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="advancedLayout">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QPlainTextEdit" name="advancedEdit"/>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QComboBox" name="helperCombo"/>
           </item>
           <item>
            <widget class="QListView" name="variablesList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="queryTab">
      <attribute name="title">
       <string>Query</string>
      </attribute>
      <layout class="QGridLayout" name="querysLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="queryScrool">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="queryAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>739</width>
            <height>435</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="queryLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="toolsLayout">
     <item>
      <widget class="QComboBox" name="operatorBox">
       <item>
        <property name="text">
         <string notr="true">AND</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">OR</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="helpLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>simpleScrool</tabstop>
  <tabstop>advancedEdit</tabstop>
  <tabstop>helperCombo</tabstop>
  <tabstop>variablesList</tabstop>
  <tabstop>queryScrool</tabstop>
  <tabstop>operatorBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>variablesList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>FilterDialog</receiver>
   <slot>variablePasteRequest(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>949</x>
     <y>612</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helperCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FilterDialog</receiver>
   <slot>helperIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>971</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>classSearchEdited(QString)</slot>
  <slot>addButtonClicked()</slot>
  <slot>operatorTextChanged(QString)</slot>
  <slot>simpleSearchEdited(QString)</slot>
  <slot>selectButtonClicked()</slot>
  <slot>unselectButtonClicked()</slot>
  <slot>validateButtonClicked()</slot>
  <slot>filterRulesChanged()</slot>
  <slot>newButtonClicked()</slot>
  <slot>limiterBoxChecked(bool)</slot>
  <slot>variablePasteRequest(QModelIndex)</slot>
  <slot>helperIndexChanged(int)</slot>
  <slot>tooltipShowRequest(QModelIndex)</slot>
 </slots>
</ui>
